libs/accounts/src/lib/accounts-table.tsx

```diff
@@ -153,31 +152,30 @@ export const AccountTable = forwardRef<AgGridReact, AccountTableProps>(
             headerName=""
             sortable={false}
             minWidth={250}
+            type="rightAligned"
             cellRenderer={({
               data,
             }: VegaICellRendererParams<AccountFields>) => {
               return data ? (
-                <div className="flex gap-2 justify-end">
-                  <Button
-                    size="xs"
+                <>
```

Comment:
chore (non blocking): lets move the break down link into the same cell so all actions are kept together